code/modules/food_and_drinks/food/snacks_pizza.dm

```diff
@@ -160,8 +160,59 @@
 	tastes = list("crust" = 1, "tomato" = 1, "cheese" = 1, "pineapple" = 2, "ham" = 2)
 	foodtype = GRAIN | VEGETABLES | DAIRY | MEAT | FRUIT | PINEAPPLE
 
+/obj/item/reagent_containers/food/snacks/pizza/arnold
+	name = "arnold pizza"
+	desc = "Hello, you've reached arnold's pizza shop. I'm not here now, I'm out killing pepperoni."
```

Comment:
however the Arnold here should be capitalized

Reply:
i just capitalized them all but made the titles improper